js/about/aboutActions.js

```diff
@@ -407,6 +407,15 @@ const aboutActions = {
       url: url,
       savePath: savePath
     })
+  },
+
+  /**
+   * Open dialog for default download path setting
+   */
+  defaultDownloadPath: function () {
```

Comment:
these no longer have to go in `aboutActions` and this file should be deprecated. Use `appActions` for now instead